README.md

```diff
@@ -320,6 +320,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Darsh Gupta](https://github.com/DarshGupta1910)
   - [Dave Bhandari](https://github.com/Davekibh)
   - [David Leal](https://github.com/Panquesito7)
+  - [David Centurion](https://github.com/Parzivalcen)
```

Comment:
Please add your name in line 322, after Dave Bhandari, and add David Leal one line below you.

Let us know if you need any help with anything! We're here to provide support. 😃

Reply:
done :)